ports/esp32/README.md

```diff
@@ -68,8 +68,8 @@ After you've cloned and checked out the IDF to the correct version, run the
 
 ```bash
 $ cd esp-idf
-$ ./install.sh       # (or install.bat on Windows)
-$ source export.sh   # (or export.bat on Windows)
+$ ./install.sh       # or (install.bat on Windows)
+$ source export.sh   # or (export.bat on Windows) or (call export.bat on Windows)
```

Comment:
Why did you move the "or" outside the parenthesis?

For the second line, might be simpler to just write "or call export.bat on Windows", no need to have both cases.

Reply:
i thought it was a little easier to read with the multiple examples, but I've changed it back closer to what you suggest.